code/modules/reagents/reagents/toxins.dm

```diff
@@ -772,3 +772,86 @@
 	if(istype(L))
 		L.adjustHalLoss(50)
 		L.adjustToxLoss(20)
+
+//Custom Chem Sir made me do to learn Chem Code
+/datum/reagent/toxin/chlorobenzalmalononitrile
+    name = "CS powder"
+    id = "chlorobenzalmalononitrile"
+    description = "A Nerve-Agent that works similar to Pepper Spray but higher in potency."
+    taste_description = "the Sun's piss"
+    taste_mult = 10
+    reagent_state = LIQUID
+    touch_met = 30
+    overdose = REAGENTS_OVERDOSE
+    color = "#545454"
+    var/agony_dose = 0.5
+    var/agony_amount = 4
+    var/discomfort_message = "<span class='danger'>You feel like the world is turning fuzzy</span>"
+/datum/reagent/toxin/chlorobenzalmalononitrile/affect_touch(mob/living/carbon/M, alien, effect_multiplier)
+    var/eyes_covered = 0
+    var/mouth_covered = 0
+    var/no_pain = 0
+    var/obj/item/eye_protection
+    var/obj/item/face_protection
+
+    var/list/protection
+    if(ishuman(M))
+        var/mob/living/carbon/human/H = M
+        protection = list(H.head, H.glasses, H.wear_mask)
+        if(H.species && (H.species.flags & NO_PAIN))
+            no_pain = 1 //TODO: living-level can_feel_pain() proc
+    else
+        protection = list(M.wear_mask)
+
+    for(var/obj/item/I in protection)
+        if(I)
+            if(I.body_parts_covered & EYES)
+                eyes_covered = 1
+                eye_protection = I.name
+            if((I.body_parts_covered & FACE) && !(I.item_flags & FLEXIBLEMATERIAL))
+                mouth_covered = 1
+                face_protection = I.name
+
+    var/message
+    if(eyes_covered)
+        if(!mouth_covered)
+            message = SPAN_WARNING("Your [eye_protection] protects your eyes from the nerve gas agent!")
+    else
+        message = SPAN_WARNING("YOUR EYES ARE BURNING!!")
+        if(mouth_covered)
+            M.eye_blurry = max(M.eye_blurry, 15)
+            M.make_dizzy(6 * effect_multiplier)
```

Comment:
You may want to increase that value or get rid of `effect_multiplier`.
Here `effect_multiplier` would be 0.223, resulting in 1.33800 `dizziness` and no visible effect.
Just so you know, `dizziness` less than 100 gives no effect and could be as high as 1000.

Reply:
Oh so that's how it goes, thank you for the clarification! This is incredibly helpful!